components/core/routes/Upload.py

```diff
@@ -0,0 +1,42 @@
+import json
+import requests
+from flask import request, Response
+from file_handler.uploader import upload_file_helper, headers
+from utils.token_utils import token_validator
+
+
+def upload_file_to_server(user_id):
+    token = token_validator(request.headers.get('token'))
+    if token is None:
+        return '{"error":"token error"}', 403
+    try:
+        form_field = request.form.get('field')
+        parent_id = request.form.get('parent_id')
+        e_type = request.args.get('type')
+        token_dict = { 'token': token }
+
+        headers.update(token_dict)
+        upload_response = upload_file_helper(form_field)
+
+        if not 'msg' in upload_response:
+            resp = Response(response='{"error":"'+ upload_response['err'] +'"}', status=500)
+
+        if upload_response['msg'] is not 'success':
+            resp = Response(response='{"error":"' + upload_response['msg'] + '"}', status=400)
+
+        file_body = "name="+upload_response['name']+"&parent_id="+parent_id
+        add_response = requests.post('http://localhost:5000/api/user/drive/'+user_id+'/add', file_body, params=e_type, headers=add_headers)
```

Comment:
can you change the URL to little dynamic as endpoint definition and hostname should be kept in constants file and then use over codebase @VibhorCodecianGupta

Reply:
oh yes I intend to do that, should i make a config file for the host and post variables then? Correct me if i'm wrong, i don't think there is one at present

Reply:
Ya, config might be the right choice. Go for it :)

Reply:
cool, i'll do that